pyfarm/agent/entrypoints/main.py

```diff
@@ -621,6 +630,27 @@ def remove_pid_file_on_exit():
 
         logger.info("pid: %s" % pid)
 
+        if not isfile(config["run_control_file"]):
+            directory = dirname(config["run_control_file"])
+            if not isdir(directory):
+                try:
+                    os.makedirs(directory)
+                except OSError:  # pragma: no cover
+                    logger.error(
+                        "Failed to create parent directory for %s",
```

Comment:
Include the exception in the error message so we can figure out what went wrong more easily.